homeassistant/components/switch/vesync.py

```diff
@@ -0,0 +1,110 @@
+"""
+Support for Etekcity VeSync switches.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.vesync/
+"""
+import logging
+import voluptuous as vol
+from homeassistant.components.switch import (SwitchDevice, PLATFORM_SCHEMA)
+from homeassistant.const import (CONF_USERNAME, CONF_PASSWORD)
+import homeassistant.helpers.config_validation as cv
+
+
+REQUIREMENTS = ['pyvesync==0.1.1']
+
+_LOGGER = logging.getLogger(__name__)
+
+API_BASE_URL = 'https://smartapi.vesync.com'
```

Comment:
Not being used

Reply:
Remnant from testing. Cleaned up in newest commit